cluster/common.sh

```diff
@@ -439,13 +450,13 @@ EOF
       ;;
     server)
       echo "Generate server certificates..."
-      echo '{"CN":"'${member_ip}'","hosts":[""],"key":{"algo":"ecdsa","size":256}}' \
+      echo "{\"CN\":""${member_ip}"",\"hosts\":[""],\"key\":{\"algo\":\"ecdsa\",\"size\":256}}" \
```

Comment:
To my eyes this looks less readable than before.  What shellcheck failure are you addressing?

Reply:
if not do that, it will report a error:
In ./cluster/common.sh line 455:
      echo '{"CN":"'${member_ip}'","hosts":[""],"key":{"algo":"ecdsa","size":256}}' \
                    ^----------^ SC2086: Double quote to prevent globbing and word splitting
when i modify this error, it will report anthor error:
In ./cluster/common.sh line 455:
      echo '{"CN":""${member_ip}"","hosts":[""],"key":{"algo":"ecdsa","size":256}}' \
           ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.
when i modify this error, it will report anthor error:

In ./cluster/common.sh line 455:
      echo "{"CN":""${member_ip}"","hosts":[""],"key":{"algo":"ecdsa","size":256}}" \
              ^-- SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                    ^---^ SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                                 ^-^ SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                                        ^--^ SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                                               ^---^ SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?
                                                                       ^--^ SC2140: Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

For more information:
  https://www.shellcheck.net/wiki/SC2140 -- Word is of the form "A"B"C" (B in...

as the same time ,This modification is in line with the shell specification, not for the convenience of reading.